src/xcode/ENA/ENA/Source/Client/Security/CoronaWarnURLSessionDelegate.swift

```diff
@@ -30,10 +30,17 @@ extension CoronaWarnURLSessionDelegate: URLSessionDelegate {
 			return
 		}
 
-		var secresult = SecTrustResultType.invalid
-		let status = SecTrustEvaluate(trust, &secresult)
-
-		if status == errSecSuccess {
+		// Apple's sample code[1] ignores the result of `SecTrustEvaluateAsyncWithError`.
+		// We consider it also safe to not check for failures within `SecTrustEvaluateAsyncWithError`
+		// that might return something different than `errSecSuccess`.
+		//
+		// [1]: https://developer.apple.com/documentation/security/certificate_key_and_trust_services/trust/evaluating_a_trust_and_parsing_the_result
+		SecTrustEvaluateAsyncWithError(trust, .main) { [weak self] trust, isValid, error in
```

Comment:
`SecTrustEvaluate` was deprecated and I replaced it with `SecTrustEvaluateAsyncWithError`, which was previously in place.

Reply:
Please keep iOS 12.5 in mind I think the new API is only available in iOS 13+

Reply:
Oh crap, I thought I covered that… 😐
Will be handled in a moment.

Reply:
Just noticed it in a quick fly over. So maybe it is covered 😅